src/Skeletor/App/Config/skeletor.yml

```diff
@@ -1,11 +1,11 @@
 config:
-    name: Skeletor CLI
+    name: 'Skeletor CLI'
```

Comment:
Hmm, i already couldn't remember changing this. But what happens, if you add a package and the `storeConfig` method runs. It exactly puts quotes around the string.